### PR TITLE
[Docs]Update Chinese document for speed benchmark

### PR DESCRIPTION
**Pull request by HanaRo**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Lack of Chinese document for benchmark.md, from issue #1361 

## Modification
Update Chinese document for benchmark.md

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/benchmarks.md [hunk 1]
```diff
@@ -1 +1,285 @@
...
+# 基准测试
+
+这里我们对 MMDetection3D 和其他开源 3D 目标检测代码库中模型的训练速度和测试速度进行了基准测试。
+
+## 配置
+
+* 硬件：8 NVIDIA Tesla V100 (32G) GPUs, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz
+* 软件：Python 3.7, CUDA 10.1, cuDNN 7.6.5, PyTorch 1.3, numba 0.48.0.
+* 模型：由于不同代码库所实现的模型种类有所不同，在基准测试中我们选择了 SECOND、PointPillars、Part-A2 和 VoteNet 几种模型，分别与其他代码库中的相应模型实现进行了对比。
+* 度量方法：我们使用整个训练过程中的平均吞吐量作为度量方法，并跳过每个 epoch 的前 50 次迭代以省略 GPU 的预热时间。
```
Comment (ZwwWayne) [inline docs/zh_cn/benchmarks.md:10]: 以消除训练预热的影响

File: docs/zh_cn/benchmarks.md [hunk 2]
```diff
@@ -1 +1,285 @@
...
+| PointPillars-car| 141           | ×         |   ×  | 140     |
+| PointPillars-3class| 107           |44     |   ×      | ×    |
+| SECOND| 40           |30     |   ×      | ×    |
+| Part-A2| 17           |14     |   ×      | ×    |
+
+## 测试细节
+
+### 为了计算速度所做的修改
+
+* __MMDetection3D__：我们尝试使用与其他代码库中尽可能相似的配置，具体配置细节见 [基准测试配置](https://github.com/open-mmlab/MMDetection3D/blob/master/configs/benchmark)。
```
Comment (ZwwWayne) [inline docs/zh_cn/benchmarks.md:28]: 相似 -> 相同

File: docs/zh_cn/benchmarks.md [hunk 3]
```diff
@@ -1 +1,285 @@
...
+
+对于模型的训练速度（样本/秒），我们将 MMDetection3D 与其他实现了相似模型的代码库进行了对比。结果如下所示，表格内的数字越大，代表模型的训练速度越快。代码库中不支持的模型使用 `×` 进行标识。
+
+|  模型  | MMDetection3D | OpenPCDet |votenet| Det3D |
+|:-------:|:-------------:|:---------:|:-----:|:-----:|
+| VoteNet | 358           | ×         |   77  | ×     |
+| PointPillars-car| 141           | ×         |   ×  | 140     |
+| PointPillars-3class| 107           |44     |   ×      | ×    |
+| SECOND| 40           |30     |   ×      | ×    |
+| Part-A2| 17           |14     |   ×      | ×    |
```
Comment (VVsssssk) [inline docs/zh_cn/benchmarks.md:22]: Regular form in .md file.
```
|        模型         | MMDetection3D | OpenPCDet | votenet | Det3D |
| :-----------------: | :-----------: | :-------: | :-----: | :---: |
|       VoteNet       |      358      |     ×     |   77    |   ×   |
|  PointPillars-car   |      141      |     ×     |    ×    |  140  |
| PointPillars-3class |      107      |    44     |    ×    |   ×   |
|       SECOND        |      40       |    30     |    ×    |   ×   |
|       Part-A2       |      17       |    14     |    ×    |   ×   |
```

File: docs/zh_cn/benchmarks.md [hunk 4]
```diff
@@ -1 +1,285 @@
...
+  ```
+
+
+  然后执行如下命令，对测试速度进行评估：
+
+  ```bash
+  python eval.py --dataset sunrgbd --checkpoint_path log_sunrgbd/checkpoint.tar --batch_size 1 --dump_dir eval_sunrgbd --cluster_sampling seed_fps --use_3d_nms --use_cls_nms --per_class_proposal
+  ```
+
+  注意，为了计算推理速度，我们对 eval.py 进行了修改。
```
Comment (ZCMax) [inline docs/zh_cn/benchmarks.md:137]: `eval.py`

File: docs/zh_cn/benchmarks.md [hunk 5]
```diff
@@ -1 +1,285 @@
...
+  ./tools/dist_train.sh configs/benchmark/hv_pointpillars_secfpn_3x8_100e_det3d_kitti-3d-car.py 8 --no-validate
+  ```
+
+* __Det3D__：在 commit [519251e](https://github.com/poodarchu/Det3D/tree/519251e72a5c1fdd58972eabeac67808676b9bb7) 版本下，使用 `kitti_point_pillars_mghead_syncbn.py` 并执行如下命令：
+
+  ```bash
+  ./tools/scripts/train.sh --launcher=slurm --gpus=8
+  ```
+
+  注意，为了训练PointPillars，我们对 train.sh 进行了修改。
```
Comment (ZCMax) [inline docs/zh_cn/benchmarks.md:212]: mind the blank:  训练 PointPillars

File: docs/zh_cn/benchmarks.md [hunk 6]
```diff
@@ -1 +1,285 @@
...
+* __OpenPCDet__：在 commit [b32fbddb](https://github.com/open-mmlab/OpenPCDet/tree/b32fbddbe06183507bad433ed99b407cbc2175c2) 版本下，执行如下命令：
+
+  ```bash
+  cd tools
+  sh scripts/slurm_train.sh ${PARTITION} ${JOB_NAME} 8  --cfg_file ./cfgs/kitti_models/pointpillar.yaml --batch_size 32  --workers 32 --epochs 80
+  ```
+
+### SECOND
+
+基准测试中的 SECOND 指在 [second.Pytorch](https://github.com/traveller59/second.pytorch) 首次被实现的 [SECONDv1.5](https://github.com/traveller59/second.pytorch/blob/master/second/configs/all.fhd.config)。在 Det3D 对 SECOND 的实现中，使用了自己实现的 Multi-Group Head，因此无法将它的速度与其他代码库进行对比。
```
Comment (ZCMax) [inline docs/zh_cn/benchmarks.md:257]: Det3D 实现的 SECOND 中，使用了自己实现的 Multi-Group Head

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**